### ISSUE TITLE
[speech-to-text] syntax errors in swift code sample and transcription does not seem to return

### ISSUE THREAD

**Issue by daniel-bolanos** [watson-developer-cloud/ios-sdk#164]
Hello, I'm posting this issue on behalf of one of our customers: 

> speech to text on github has some syntax errors in swift code sample and transcription does not seem to return transcription in text (response) below….has the backend service changed/not working with swift sdk since published on github?  
 
 
https://github.com/watson-developer-cloud/ios-sdk?cm_mc_uid=67919448948414527764977&cm_mc_sid_50200000=1452873153
 
```swift
if let data = data {
  sttService.transcribe(data , format: .WAV, oncompletion: {
    response, error in
    // code here
  }
}
```

**Comment by lucasdamiani** [comment]
I had this issue when transcribing a portuguese audio sample to text. I've fixed it by specifying the an audio format compatible to the audio I've recorded. In my case, I was using WAV. Perhaps it would be good to automatically map the format based on the file extension. I figure out that when you specify the incorrect audio format, the completionHandler isn't called.

**Comment by glennrfisher** [comment]
Closed in favor of #168.
